src/components/Footer/socialMediaLinks/SocialMediaLinks.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { SocialMediaIconsReact } from "social-media-icons-react";
+
+function SocialMediaLinks({ icon, backgroundColor }) {
+  const links = [
+    {
+      id: 1,
+      icon: "twitter",
+      backgroundColor: "rgba(26,166,233,1)",
+    },
+    {
+      id: 2,
+      icon: "facebook",
+      backgroundColor: "rgba(26,166,233,1)",
+    },
+  ];
+  let bgColor = "rgba(26,166,233,1)";
```

Comment:
Do we need `let`? Can this not be `const` instead?

Reply:
Yeah right, will make it const.